docs/fee-market-strategy.md

```diff
@@ -3,54 +3,32 @@ Fee Market Strategy
 
 ## Relay strategy
 
-In Darwinia <> Crab and Pangolin <> Pangoro bridge. All relayers can decide
-whether to relay a transaction. Decide on your role (Assigned Relayer or Not
-Assigned Relayer). Different roles will have different benefits.
+For s2s bridge (such as Darwinia<>Crab). All relayers can decide whether to
+relay a transaction. Decide on your role (Assigned Relayer or Not Assigned
+Relayer). Different roles will have different benefits.
 
 You can refer to this picture:
 
-![Fee market rewards strategy](./assets/fee-market-rewards-strategy.png)
+|       Title       |                                      Image                                       |
+|:-----------------:|:--------------------------------------------------------------------------------:|
+| Rewards Pie Chart | ![Fee market rewards strategy v2](./assets/fee-market-rewards-strategy-v2-1.png) |
+|  Reward Process   | ![Fee market rewards strategy v2](./assets/fee-market-rewards-strategy-v2-2.png) |
 
-So if you have good ideas, you can modify it yourself.
+The more details you can
+read [What Can A Cross-Chain Message Relayer Earn?](https://medium.com/darwinianetwork/what-can-a-cross-chain-message-relayer-get-ffb683b689cb)
```

Comment:
Suggest moving the photos to the medium link docs, so we only need to maintain it in one place.

The photos from Aki still need to be polished, it's not finished.

Reply:
Agree with this. I think we just wait for fee-market documents to be published. and there just put a link.
cc @wuminzhe, You can see if this photo helps you.